frontend/src/components/Table.tsx

```diff
@@ -11,6 +11,7 @@ import {
 interface Props {
   selection: "multiple" | "none";
   initialSortByField?: string;
+  asc?: boolean;
```

Comment:
Nit picking: To follow the same naming convention of the other Prop, can you call this property `initialSortAscending?: boolean`?